### ISSUE TITLE
Add arn as an attribute for aws_vpc

### ISSUE THREAD

**Issue by tdmalone** [terraform-providers/terraform-provider-aws#5027]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

```
Terraform v0.11.7
+ provider.aws v1.25.0
```

### Affected Resource(s)

* [aws_vpc](https://www.terraform.io/docs/providers/aws/r/vpc.html)
* [aws_default_vpc](https://www.terraform.io/docs/providers/aws/r/default_vpc.html)
* [aws_vpc](https://www.terraform.io/docs/providers/aws/d/vpc.html) (data source)

### Terraform Configuration Files

```hcl
# Ideal.
output "services_arn" {
  value = "${aws_vpc.services.arn}"
}

# Current workaround.
output "services_arn" {
  value = "arn:aws:ec2:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:vpc/${aws_vpc.services.id}"
}
```

### Expected Behavior

Expected (or hoped) there would be an ARN attribute exported for the VPC ;)

### Actual Behavior

```
Error: Error running plan: 1 error(s) occurred:

* output.services_arn: Resource 'aws_vpc.services' does not have attribute 'arn' for variable 'aws_vpc.services.arn'
```

### Important Factoids

As a use case scenario, the VPC ARN is needed when using IAM policy conditions `ec2:Vpc`, `ec2:AccepterVpc`, and `ec2:RequesterVpc`, as seen in the table [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/list_amazonec2.html#amazonec2-policy-keys).

Also, similar to #5028.

**Comment by MattMcNam** [comment]
Interested in working on this in the next day or so

**Comment by bflad** [comment]
The `arn` attribute has been added to the `aws_vpc` resource, `aws_default_vpc` resource, and `aws_vpc` datasource via #5300 and will release with version 1.29.0 of the AWS provider, likely middle of this week. 👍

**Comment by bflad** [comment]
This has been released in [version 1.29.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1290-july-26-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
